etc/profile-m-z/telegram.profile

```diff
@@ -47,4 +47,10 @@ private-dev
 private-etc alsa,alternatives,ca-certificates,crypto-policies,fonts,group,ld.so.cache,localtime,machine-id,os-release,passwd,pki,pulse,resolv.conf,ssl,xdg
 private-tmp
 
+dbus-user filter
+dbus-user.own org.telegram.desktop
```

Comment:
If `busctl --user list | grep org.telegram.desktop` does not show it while you run telegram, we can remove it.

Reply:
yea. I thought gnome or any other de has it.